src/components/command-bar/index.tsx

```diff
@@ -175,7 +172,10 @@ const CommandBarProvider = ({ children }: CommandBarProviderProps) => {
 				<Command.Dialog isOpen={state.isOpen} onDismiss={toggleIsOpen}>
 					<Command.Header />
 					<Command.Body />
-					<Command.Footer />
+					{
+						// TODO(kevinwang): This conditional needs an abstraction
```

Comment:
🎏 Flagging this unresolved `TODO` - hot take we could probably just leave this as-is and figure out a more robust solution later, if we want!

Reply:
ACK - I'm going to take no action for now